### PR TITLE
Install bazel 3.1.0 on kubekins-e2e images

### PR DESCRIPTION
**Pull request by chaodaiG**

### CODE REVIEW

File: images/kubekins-e2e/variants.yaml [hunk 1]
```diff
@@ -16,8 +16,8 @@ variants:
     CONFIG: master
```
Comment (fejta) [inline images/kubekins-e2e/variants.yaml:16]: please create a test-infra variant, do not corrupt the kubernetes/kubernetes variant

Comment (chaodaiG) [reply images/kubekins-e2e/variants.yaml:22]: Done.

**Comment by fejta** [comment]
why are we using kubekins rather than krte?

**Comment by chaodaiG** [comment]
It's symlinked I believe, this file is used by both images

**Comment by fejta** [comment]
THis LGTM but I don't see how krte has anything to do with anything.

I can't actually identifying anything using krte (including this PR). Should we just delete this image?

**Comment by fejta** [comment]
> I can't actually identifying anything using krte (including this PR). Should we just delete this image?

Ah, no it looks like things that do not use bazel sometimes use this image: https://github.com/search?q=org%3Akubernetes+krte&type=code

Can you remove the references to krte here? It is unrelated to your task.

**Comment by chaodaiG** [comment]
Sounds good, updated PR title

**Review by fejta — APPROVED** [review]

**Pull request merged**